frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlertsRulesDetailsPage.tsx

```diff
@@ -2,19 +2,31 @@ import * as React from 'react';
 import * as _ from 'lodash';
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { match as RMatch } from 'react-router';
-import { alertingLoaded, alertingSetRules } from '@console/internal/actions/observe';
-import { usePrometheusRulesPoll } from '@console/internal/components/graphs/prometheus-rules-hook';
+import { useResolvedExtensions } from '@console/dynamic-plugin-sdk';
+import {
+  AlertingRulesSourceExtension,
+  isAlertingRulesSource,
+} from '@console/dynamic-plugin-sdk/src/extensions/alerts';
```

Comment:
```suggestion
import {
  AlertingRulesSourceExtension,
  isAlertingRulesSource,
  useResolvedExtensions,
} from '@console/dynamic-plugin-sdk';
```
Maybe combine the `dynamic-plugin-sdk` imports?

Reply:
Not all packages are exported in the main `@console/dynamic-plugin-sdk` so a full path is needed

Reply:
I added the alerts to the main package export in the SDK